README.md

```diff
@@ -1,27 +1,31 @@
 # Discovery Bib/Item Poster
 
-Reads bibs or items from stream, posts to bib or item service.
+This lambda should be deployed to two different lambdas.
+One to handle bibs, another for items. It reads those bibs or items from stream, then posts them to the bib or item service.
 
 ## Setup
 
-```
+1.  ```
 npm install
 npm install -g node-lambda
-node-lambda setup
+cp example.env .env
+cp deploy.example.env deploy-bib.env
+cp deploy.example.env deploy-item.env
 ```
 
```

Comment:
That's what I thought. Cool, yeah would have been nice if they had a way to specify which .env we want to use during deployment.